src/index.js

```diff
@@ -332,6 +333,8 @@ function addSlotToRoot (root, slot) {
   }
 
   rootToSlotMap.get(root)[slotName] = slot;
+  !slotToRootMap.has(slot) && slotToRootMap.set(slot, root);
```

Comment:
We should probably try to keep the coding styles similar, even if they're separate projects. We maintain both and the less cognitive overhead in doing so, the better.